docs/source/evaluation_suite.mdx

```diff
@@ -0,0 +1,48 @@
+# Creating an EvaluationSuite
```

Comment:
I think it would be good to give instructions how to add a new Suite on the hub (for people who don't know how). For metrics there is a small CLI using Cookiecutter where you specify the name of the metric and it creates the spaces, clones it locally and adds template files. Similar to the metric modules we could also add 
- README: document the goal and limitations of the suite
- app.py: a simple gradio demo or so that parses the Suite.py/README and displays useful information
If you want I could help with that. I think reducing friction to create a new suite to the minimum will maximise adoption. Happy to do it in a follow up PR but I think it would be great to have with the release/announcement. What do you think?

Reply:
Sure, sounds good, having a README and/or template seems broadly useful!

I've added some more instructions in evaluation_suite.mdx as well.